### PR TITLE
Add option to customize generated cell keys

### PR DESCRIPTION
**Pull request by Artsu**
Added option to customize each cell's key as sometimes the camelcase library does a poor job guessing the correct key. For example special characters and letters outside ascii might get switched to underscore.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -45,6 +45,10 @@ The plugin is based on the [node-sheets](https://github.com/urbancups/node-sheet
     // The `mapValue(value, key, cell)` function lets you map each cell's value to your need.
     // For example, it can be used to ensure that all strings are trimmed:
     mapValue: value => typeof value === "string" ? value.trim() : value
+    
+    // The `mapKey(key)` function lets you map each cell's key as sometimes the camelcase library does a poor job guessing
+    // the correct key. For example special characters and letters outside ascii might get switched to underscore.
+    mapValue: key => key === 'ääöö' ? 'aaoo' : key
```
Comment (sondrele) [inline README.md]: This should be `mapKey` :smiley:

**Review by sondrele — CHANGES_REQUESTED** [review]
Thanks for the contribution! Will release once the typo in the readme has been fixed :smiley:

File: README.md (hunk 1 above)
Comment (Artsu) [reply README.md]: Yeah it should 😅 Fixed it now!

**Comment by sondrele** [comment]
@Artsu Sorry for the late reply, I've been on vacation and quite busy with other things lately.
It also turns out I already released version 2 of this source plugin a while ago, but forgot to update the master branch (I had a vague resemblance of working on something like this, but I completely missed it while vacationing). Thus, your changes can be solved by configuring and implementing the `mapNode` option instead.
Really sorry for the inconvenience and for wasting your time! :(

**Pull request closed without merging**